apps/studio/src/constants/formBuilder.ts

```diff
@@ -4,6 +4,7 @@ export const JSON_FORMS_RANKING = {
   DropdownControl: 2,
   IntegerControl: 4,
   TextControl: 1,
+  ObjectControl: 2,
```

Comment:
What's this ranking for?

Reply:
It's a JSONForms concept, mainly to determine the order in which the renderers are chosen.